README.md

```diff
@@ -4,6 +4,36 @@
 
 This repository contains the source for the Google-maintained Tomcat [docker](https://docker.com) image.
 
+# Building the image
```

Comment:
What if you move this to DEVELOPING.md and add a line here that says "This image has not yet been released. For instructions on building from source, see DEVELOPING.md"